README.md

```diff
@@ -79,7 +65,10 @@ For information about the HAL and how to use it in your own projects, please ref
 [esp32-s2]: https://www.espressif.com/sites/default/files/documentation/esp32-s2_technical_reference_manual_en.pdf
 [esp32-s3]: https://www.espressif.com/sites/default/files/documentation/esp32-s3_technical_reference_manual_en.pdf
 
-### Low-Power Cores
+### [esp-lp-hal]
+
+Implements a number of the traits defined in [embedded-hal](https://github.com/rust-embedded/embedded-hal) for the low-power RISC-V coprocessors found on the ESP32-C6, ESP32-S2, and ESP32-S3 from Espressif.
```

Comment:
It does a bit more than that too, maybe we can expand on the goals of the hal, i.e that it runs on the LP core itself etc.

Reply:
Added some more information, hopefully didn't miss anything relevant :D